src/main.go

```diff
@@ -51,6 +51,7 @@ func main() {
 	c.AddFunc("@every 5m", history.BackupState)
 	c.AddFunc("@every 20m", options.UpdateRepos)
 	c.AddFunc("@every 2m", controllers.HealthCheck)
+	c.AddFunc("@monthly", options.CleanupDanglingImages)
```

Comment:
Please put it in controllers. This is making a change to the VMs, so don't put it in options.